source/language/types.rst

```diff
@@ -290,9 +291,10 @@ Duration
 ~~~~~~~~
 
 We introduce a ``duration`` type to express timing.
-Durations are positive numbers with a unit of time. ``ns, μs, ms, s`` are used for SI time
+Durations are numbers with a unit of time. ``ns, μs, ms, s`` are used for SI time
```

Comment:
We should also allow "us" as a synonym for "μs"

Reply:
Added in https://github.com/Qiskit/openqasm/pull/268/commits/6088c0267f31a9a56f6693e4fb58609f41ed46e2